workflows/argo/training.yaml

```diff
@@ -18,7 +18,7 @@ spec:
         parameters:
           - name: training_config
           - name: training_data_config
-          - name: validation_data_config
+          - {name: validation_data_config, value: " "}
```

Comment:
I made this optional to reflect that the training script does not need this as an input, and often times users may not provide a separate validation config to the `train-diags-prog` workflow in addition to the test data config. Previously the same data config was used both as train time validation data and offline evaluation, but that is no longer possible.

Reply:
Thanks this will be useful. The RF training script wastes time preprocessing the validation data which it doesn't end up using during training.